src/renderers/dom/stack/client/wrappers/ReactDOMInput.js

```diff
@@ -174,14 +171,17 @@ var ReactDOMInput = {
     var node = ReactDOMComponentTree.getNodeFromInstance(inst);
     var value = props.value;
     if (value != null) {
-
-      // Cast `value` to a string to ensure the value is set correctly. While
-      // browsers typically do this as necessary, jsdom doesn't.
-      var newValue = '' + value;
-
-      // To avoid side effects (such as losing text selection), only set value if changed
-      if (newValue !== node.value) {
-        node.value = newValue;
+      if (value === 0) {
```

Comment:
@nhunzaker can we check `node.value` here too and see if its `"0.0"`? Since `0.0 === 0` I think we're going to need to rely on checking the stringified value of the node's value.